google-newtab/css/main.css

```diff
@@ -0,0 +1,13 @@
+body{
+    display: inline-block;
+    align-content: center;
```

Comment:
I could be wrong, but I believe align-content can only be used for flex-box.  Since you are using display: inline-block, i do not believe it will have any affect on it.  If you are trying to align to the center, you could set it to flexbox and use justify-content: center and align-items: center.  Or text-align: center may work.

Reply:
Yeah, I had them with flexbox first it is why it is there. I just forgot to delete it.